sdk/protocol-plugins/scripts/plugin-template/{{name}}ProtocolPlugin.ts.mustache

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
+  Percentage,
+  TokenAmount,
+  TokenSymbol,
+  Price,
+  RiskRatio,
+  Address,
+  Position,
+  ChainId,
+  ChainFamilyName,
+  valuesOfChainFamilyMap,
+} from '@summerfi/sdk-common/common'
+import { PoolType, ProtocolName } from '@summerfi/sdk-common/protocols'
+import { SimulationSteps } from '@summerfi/sdk-common/simulation'
+import type { {{name}}PoolId } from '@summerfi/sdk-common/protocols'
+import { stringToHex, getContract } from 'viem'
+import { BigNumber } from 'bignumber.js'
+import { z } from 'zod'
+import { {{name}}PaybackWithdrawActionBuilder } from './builders/{{name}}PaybackWithdrawActionBuilder.ts'
```

Comment:
from './builders/{{name}}PaybackWithdrawActionBuilder' (without the .ts extension)